src/routes/RouteNames.js

```diff
@@ -28,6 +28,8 @@ const RouteNames = {
   addmenu: '/addmenu',
   editmenu: '/editmenu',
   orderHistories: '/orderHistories',
+  balanceSheet: '/balanceSheet',
+  editBalanceSheet: '/editBalanceSheet',
```

Comment:
please try to make sure that all routes in the project are hyphenated like edit-balance-sheet